pytest_catchlog.py

```diff
@@ -118,10 +129,20 @@ def __init__(self, config):
         The formatter can be safely shared across all handlers so
         create a single one for the entire test session here.
         """
+        self.config = config
```

Comment:
Do you really need it? Better pass it as argument to `self.configure_console_handler`.